### PR TITLE
Fix jitter fix

### PR DESCRIPTION
**Pull request by jdelapla**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/source/PeerConnection/JitterBuffer.c [hunk 1]
```diff
@@ -138,14 +138,13 @@ STATUS jitterBufferPush(PJitterBuffer pJitterBuffer, PRtpPacket pRtpPacket, PBOO
...
-                    {
+                    if ((pRtpPacket->header.sequenceNumber >= (MAX_UINT16 - (MAX_OUT_OF_ORDER_PACKET_DIFFERENCE - pJitterBuffer->headSequenceNumber))) ||
+                       (pJitterBuffer->headSequenceNumber > pRtpPacket->header.sequenceNumber)) {
                         pJitterBuffer->headSequenceNumber = pRtpPacket->header.sequenceNumber;
                     }
                 }
-                else if((pJitterBuffer->headSequenceNumber - MAX_OUT_OF_ORDER_PACKET_DIFFERENCE) <= pRtpPacket->header.sequenceNumber)
-                {
+                else if ((pRtpPacket->header.sequenceNumber >= pJitterBuffer->headSequenceNumber - MAX_OUT_OF_ORDER_PACKET_DIFFERENCE) && 
+                        (pRtpPacket->header.sequenceNumber < pJitterBuffer->headSequenceNumber) {
```
Comment (hassanctech) [inline src/source/PeerConnection/JitterBuffer.c:147]: Is this something we could add an easy test for?

**Comment by disa6302** [comment]
Clang format in CI is broken. @jdelapla , please make sure to clang format your file using the instructions [here](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#clang-checks). YTO format you will need to run `./scripts/clang-format.sh -f <filename>`

**Review by disa6302 — APPROVED** [review]

**Comment by hassanctech** [comment]
@jdelapla It doesn't compile looks like there is a ) or } mismatch, also if possible please add a unit test since it's a bug fix for an edge case.

Also CodeQL failed (which I think just tries to do a build so I think once you resolve the compile issue, it should succeed).

File: src/source/PeerConnection/JitterBuffer.c (hunk 1 above)
Comment (hassanctech) [reply src/source/PeerConnection/JitterBuffer.c:147]: Need missing `)`

**Review by hassanctech — APPROVED** [review]
